LICENSE

```diff
@@ -1,41 +1,32 @@
-Shopsys Community License
```

Comment:
we should probably edit also `utils/license-acknowledgements-generator/template.md` and `open-source-license-acknowledgements-and-third-party-copyrights.md`

Third line contains `Shopsys Platform is licensed under the terms of the [Shopsys Community License](./LICENSE).`

Reply:
Nice catch, fixed